modules/kubernetes_node_pools/variables.tf

```diff
@@ -3,20 +3,22 @@ variable "node_pools" {
 
   default = [
     {
-      name           = "additional-pool"
-      min_node_count = 1
-      max_node_count = 1
-      version        = "1.10.6-gke.2"
-      image_type     = "COS"
-      machine_type   = "n1-standard-1"
-      preemptible    = false
-      tags           = "additional-pool worker"
+      name               = "additional-pool"
+      initial_node_count = 1
+      min_node_count     = 1
+      max_node_count     = 1
```

Comment:
maybe, just an idea: The purpose of default values is to give the best setup, isn't it better to have more nodes recommended way? in this way, I would suggest to increase it at least to 2 in `max_node_count`

it would be breaking change, though...

Reply:
I think I cannot make it as totally non-breaking change, as I am removing the default node pool. Also please see the default value here: https://github.com/russmedia/terraform-google-kubernetes-cluster/pull/11/files#diff-c9ac8098c5ea9d3e6a9a596ff0c512a4R60